sentry_sdk/integrations/aws_lambda.py

```diff
@@ -100,7 +101,9 @@ def sentry_handler(event, context, *args, **kwargs):
                     timeout_thread.start()
 
             try:
-                return handler(event, context, *args, **kwargs)
+                transaction = Transaction(op="aws_lambda", name=context.function_name)
+                with hub.start_transaction(transaction):
+                    return handler(event, context, *args, **kwargs)
             except Exception:
```

Comment:
Please put the exception block inside of start_transaction.

Also setting `scope.transaction` should no longer be necessary.

The same comments apply to GCP

Reply:
Sure, made this change.